qiskit/providers/ibmq/experiment/experimentservice.py

```diff
@@ -317,6 +317,8 @@ def analysis_results(
             experiment_id: Optional[str] = None,
             result_type: Optional[str] = None,
             quality: Optional[List[Tuple[str, Union[str, ResultQuality]]]] = None,
+            tags: Optional[List[str]] = None,
+            tags_operator: Optional[str] = "OR",
```

Comment:
There should be a `features` release note for this since it's a new feature.

Reply:
Updated!